packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -265,6 +275,43 @@ function notifyDebuggerSessionFileChanged(): void {
   }
 }
 
+async function notifyIsvAuthChanged() {
+  if (
+    vscode.workspace.workspaceFolders instanceof Array &&
+    vscode.workspace.workspaceFolders.length > 0
+  ) {
+    const forceConfig = await new ForceConfigGet().getConfig(
+      vscode.workspace.workspaceFolders[0].uri.fsPath,
+      'isvDebuggerSid',
```

Comment:
Thoughts on using constants for these? They're in three packages now. It makes more sense to put the constant in a debugger package, but core doesn't depend on debugger. The only common package is utils.